appengine/redis/server.js

```diff
@@ -0,0 +1,55 @@
+var redis = require("redis");
+var http = require('http');
+var util = require('util');
+var nconf = require('nconf');
+
+// read in keys and secrets.  You can store these in a variety of ways.  
+// I like to use a keys.json  file that is in the .gitignore file, 
+// but you can also store them in environment variables
+nconf.argv().env().file('keys.json');
+
+// Connect to a redis server provisioned over at 
+// redisLabs.  See the README for more info. 
+var client = redis.createClient(
+    nconf.get("redisPort"),
+    nconf.get("redisHost"), 
+    {
+        auth_pass: nconf.get("redisKey"), 
+        return_buffers: true
+    }
+).on("error", function (err) {
+    logger.error("ERR:REDIS: " + err);
```

Comment:
Running `npm start` without valid (or default, see above comments) redis connection values causes this errback to be called, revealing this error:

```
jdobry-macbookpro:redis jdobry$ npm start

> appengine-redis@1.0.0 start /Users/jdobry/projects/nodejs-docs-samples/appengine/redis
> node server.js

started web process

/Users/jdobry/projects/nodejs-docs-samples/appengine/redis/server.js:21
    logger.error("ERR:REDIS: " + err);
    ^
ReferenceError: logger is not defined
    at RedisClient.<anonymous> (/Users/jdobry/projects/nodejs-docs-samples/appengine/redis/server.js:21:5)
    at RedisClient.emit (events.js:95:17)
    at RedisClient.on_error (/Users/jdobry/projects/nodejs-docs-samples/appengine/redis/node_modules/redis/index.js:196:10)
    at Socket.<anonymous> (/Users/jdobry/projects/nodejs-docs-samples/appengine/redis/node_modules/redis/index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:441:14
    at process._tickCallback (node.js:448:13)
```